### DISCUSSION TITLE
Use API in paid app

### DISCUSSION THREAD

**Discussion by kuznec2222** [open-meteo/open-meteo#120] [General]
Hi! Can I use this API in paid app? Currently I use Dark Sky service but it will close soon. My app use about 40.000 requests per month.

**Comment by patrick-zippenfenig** [comment]
Hi. It is currently not officially possible. In the future there should be affordable SaaS options for commercial use available with SLA, support, etc. Right now, it is tolerated and with less than 40k per month you should also not be blocked. Please consider sponsoring the project in the meantime until commercial offerings are available!
